changelogs/unreleased/gh-5799-add-csw-to-fiber-object.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua/fiber
+
+* Introduce `fiber_object:csw()` to get `csw` from fiber
```

Comment:
Please, add a reference to the ticket in the form `(gh-5799)`. Also you don't need multiple changelog files for one feature.

Reply:
There are 2 features (new functions in `fiber_object` and setting `fiber->csw=0`) in this patch so, I think, there should be 2 changelog files.

Added reference to the ticket and reduced number of changelog files from 3 to 2.